grpc-gateway/subscription/convertToSubjects_test.go

```diff
@@ -4,34 +4,75 @@ import (
 	"sort"
 	"testing"
 
+	"github.com/plgd-dev/hub/v2/grpc-gateway/pb"
 	isEvents "github.com/plgd-dev/hub/v2/identity-store/events"
 	"github.com/plgd-dev/hub/v2/resource-aggregate/commands"
 	"github.com/plgd-dev/hub/v2/resource-aggregate/cqrs/utils"
 	"github.com/plgd-dev/hub/v2/resource-aggregate/events"
-	kitStrings "github.com/plgd-dev/kit/v2/strings"
 	"github.com/stretchr/testify/require"
 )
 
 func TestConvertToSubjects(t *testing.T) {
-	resourceID := commands.NewResourceID("a", "/light/2").ToUUID()
+	resourceID := commands.NewResourceID("a", "/light/2")
 	type args struct {
-		owner             string
-		filterDeviceIDs   kitStrings.Set
-		filterResourceIDs kitStrings.Set
-		bitmask           FilterBitmask
+		owner string
+		req   *pb.SubscribeToEvents_CreateSubscription
 	}
 	tests := []struct {
 		name string
 		args args
 		want []string
 	}{
 		{
-			name: "empty",
+			name: "all",
+			args: args{
+				req: &pb.SubscribeToEvents_CreateSubscription{},
+			},
+			want: []string{
+				isEvents.ToSubject(isEvents.PlgdOwnersOwner+".>", isEvents.WithOwner("")),
+			},
 		},
 		{
-			name: "all",
+			name: "all - DeviceIdFilter *",
+			args: args{
+				req: &pb.SubscribeToEvents_CreateSubscription{
+					DeviceIdFilter: []string{"*", "*"},
```

Comment:
Why `[]string{"*", "*"}` instead of just `[]string{"*"}`? Testing deduplication?

(Same for the following three test cases.)

Reply:
Yes, I want to check if they are deduplicated.